earth2studio/statistics/crps.py

```diff
@@ -0,0 +1,190 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+from earth2studio.utils.coords import handshake_coords, handshake_dim
+from earth2studio.utils.type import CoordSystem
+
+
+class crps:
+    """
+    Compute the Continuous Ranked Probably Score (CRPS).
+
+    Uses this formula
+        # int [F(x) - 1(x-y)]^2 dx
+
+    where F is the emperical CDF and 1(x-y) = 1 if x > y.
+
+    This method is more memory efficient than the kernel method, and uses O(n
```

Comment:
"This method is more memory efficient than the kernel method"

Seems repeated with whats below, is this comment about kernel method needed here?